### PR TITLE
Packages should manage their own framework name

### PR DESCRIPTION
**Pull request by jsancio**
The latest version of Cosmos is going to force packages to manage their
own framwwork name. This change makes it possible for us to change
Cosmos without breaking all of the packages.

### CODE REVIEW

File: repo/packages/M/marathon-lb/0/marathon.json.mustache [hunk 1]
```diff
@@ -51,7 +51,6 @@
   "uris": [ "{{marathon-lb.template-url}}" ],
   {{/marathon-lb.template-url}}
   "env": {
-    "FRAMEWORK_NAME": "{{marathon-lb.framework-name}}",
```
Comment (jsancio) [inline repo/packages/M/marathon-lb/0/marathon.json.mustache]: marathon-lb doesn't have a property called `marathon-lb.framework-name`

File: repo/packages/M/marathon-lb/1/marathon.json.mustache [hunk 2]
```diff
@@ -55,7 +55,6 @@
   "uris": [ "{{marathon-lb.template-url}}" ],
   {{/marathon-lb.template-url}}
   "env": {
-    "FRAMEWORK_NAME": "{{marathon-lb.framework-name}}",
```
Comment (jsancio) [inline repo/packages/M/marathon-lb/1/marathon.json.mustache]: marathon-lb doesn't have a property called `marathon-lb.framework-name`

**Comment by BenWhitehead** [comment]
This looks okay to me, we probably want to run a describe agains each of these to make sure they render valid json.

**Comment by jsancio** [comment]
Yes. I am going to work on that today.

**Comment by jsancio** [comment]
@BenWhitehead Done. Tested that `dcos package describe --app --render <package-name>` works for all of the packages.

The only package for which it doesn't work is `datadog` but that package has require properties that don't have default. I have tested that it work when `--option` is passed with those properties.

**Comment by jsancio** [comment]
@BenWhitehead This is ready to merge.

**Pull request merged**